tests/integration/test_ibootbar_agent_integration.py

```diff
@@ -27,6 +26,11 @@
     "ibootbarAgent",
     args=["--log-dir", "./logs/"],
 )
+run_agent_acq = create_agent_runner_fixture(
+    "../agents/ibootbar/ibootbar.py",
+    "ibootbarAgent",
+    args=["--log-dir", "./logs/", "--mode", "test"],
+)
```

Comment:
This isn't used, and also sets `--mode` to `test` instead of `acq`. Should a test be using it?

Reply:
Ah yeah that was left over from something else I was trying, I'll delete and commit